pycompwa/data/kinematics.py

```diff
@@ -0,0 +1,44 @@
+"""A façade to the :class:`~.HelicityKinematics` class.
+
+.. todo:
+    These functions are basically façades to the :mod:`pycompwa.ui` so don't
+    belong in the :mod:`pycompwa.data` module.
+"""
```

Comment:
This stuff has go to the ui.

Reply:
Have a fix ready for this PR, but this requires https://github.com/ComPWA/ComPWA/pull/311

Reply:
https://github.com/ComPWA/pycompwa/pull/78/commits/da5ec56485ea60ed1148b9aa1694ac689a2b8ed0